Compiler/FunctionCompiler.py

```diff
@@ -749,6 +749,35 @@ def compile_if(self):
         self.increase_stack_pointer(amount=2)
         inside_if_code = self.compile_scope()
 
+        if self.parser.next_token().type == Token.IF:
```

Comment:
I think it's more correct to check if we have brackets or not, and based on that call either compile_statement or compile_scope
Because this current implementation will only work for another if statement. And we want to support all statements. E.g
if (a==b) {} else x=1;
Won't work with this implementation

The same thing should be after the "if"

Reply:
Please see https://github.com/elikaski/BF-it/issues/8#issuecomment-784603326
Due to indentation being needed then you would have to implement that. Edits are allowed on this pr.

Reply:
Isn't it better to just merge this and implement bracket-less scopes later.

Reply:
I'm sorry, I don't understand
Neither C nor this compiler look at indentation

Reply:
After trying a real compiler. The compiler seems to only run the first statement.

[Here](https://tio.run/##S9ZNT07@/185My85pzQlVcGmuCQlM18vw46LKzOvRCE3MTNPQ1OhmksBCEACJanFJQq2CkbWXBChNA2IiK2CoSZYBAQKSkuKNZQ8UnNy8pU0rdGEFcLzi3JSQOKkmYCuu/b/fwA)

Reply:
Since there exists multiple places where you don't need brackets such as for, if, else, while. This involves more than just else if. And can be fixed/implemented in another pull request.

Reply:
I'd rather have one commit that handles all of the cases you mentioned (for, wile, if, else) than just have it work for if